src/app/[locale]/mindmap/components/MindmapButtonsPanel.tsx

```
@@ -0,0 +1,93 @@
+import VerticalLayoutButton from '@/components/mindmap/button/VerticalLayoutButton';
+import MindmapLayoutButton from '@/components/mindmap/button/MindmapLayoutButton';
+import DeleteMindmapButton from '@/components/mindmap/button/DeleteMindmapButton';
+import { useMindMapContext } from '../context/MindMapContext';
+import { Expand, Save } from 'lucide-react';
+import { Toggle } from '@/components/ui/toggle';
+import ImportMindmapButton from '@/app/[locale]/mindmap/components/buttons/ImportMindmapButton';
+// import ImportMindmapButton from './buttons/ImportMindmapButton';
+
+// import { Toggle } from "@/components/ui/toggle"
+
+const MindmapButtonsPanel = ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)